website/docs/sdk-reference/dotnet.md

```diff
@@ -140,12 +140,49 @@ You can close all open clients at once using the `ConfigCatClient.DisposeAll()`
 
 ```csharp
 User userObject = new User("#UNIQUE-USER-IDENTIFIER#");  // Optional User Object
-var value = client.GetValue("keyOfMySetting", false, userObject);
+var value = client.GetValue("keyOfMyFeatureFlag", false, userObject);
 ```
 
 ```csharp
 User userObject = new User("#UNIQUE-USER-IDENTIFIER#"); // Optional User Object
-var value = await client.GetValueAsync("keyOfMySetting", false, userObject);
+var value = await client.GetValueAsync("keyOfMyFeatureFlag", false, userObject);
+```
+
+:::caution
+Please keep in mind that you need to supply an argument for the `defaultValue` parameter (more precisely, the `T` generic type parameter)
```

Comment:
When using the defaultValue parameter, it is important to provide an argument corresponding to the kind of feature flag or setting you are evaluating. The type of the defaultValue should align with the specific kind of feature flag or setting, as indicated in the following table. This ensures that the evaluation runs correctly: